core/spec/models/spree/moss_taxes_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'spec_helper'
+
+describe Spree::TaxRate, :type => :model do
+  context '#adjust with masala' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.